recipes/co/all/conanfile.py

```diff
@@ -12,6 +11,9 @@ class CoConan(ConanFile):
     license = "MIT"
     description = "A go-style coroutine library in C++11 and more."
     topics = ("co", "coroutine", "c++11")
+
+    deprecated = "cocoyaxi"
```

Comment:
I'm wondering: should there be `provides` also?

Reply:
I'm not familar with conan. So what I should do now?

Reply:
> I'm wondering: shouldn't there be `provides` also?

I have added cocoyaxi in #8930

Reply:
For co 2.0.2, the sha256 sum should be 5dbb90e18627ebe563a6a437e8e28329a6bfc07f266718318bd3adf26027d156